.github/workflows/review-site.yml

```diff
@@ -110,12 +110,13 @@ jobs:
         run: |
           terminus -y -n connection:set ${{ secrets.DEMO_SITE_NAME }}.dev sftp
           terminus -y -n drush ${{ secrets.DEMO_SITE_NAME }}.dev -- \
-          site-install az_quickstart install_configure_form.update_status_module='array(FALSE,FALSE)' \
+          site:install az_quickstart install_configure_form.enable_update_status_module=NULL \
```

Comment:
From https://drushcommands.com/drush-8x/core/site-install/
```drush site-install standard install_configure_form.enable_update_status_module=NULL install_configure_form.enable_update_status_emails=NULL
```

Reply:
That option is only evaluated if `enable_update_status_module` is TRUE:
https://git.drupalcode.org/project/drupal/-/blob/9.2.x/core/lib/Drupal/Core/Installer/Form/SiteConfigureForm.php#L315-323